src/python/national_voter_file/tests/faker_data.py

```diff
@@ -12,7 +12,8 @@
 
 
 TEST_DATA_DIR = os.path.join(os.path.dirname(__file__), 'test_data')
-TEST_STATES = ['de', 'co', 'fl', 'mi', 'nc', 'ny', 'oh', 'ok', 'pa', 'ut', 'wa']
+
+TEST_STATES = ['de', 'co', 'fl', 'mi', 'nc', 'ny', 'oh', 'ok', 'pa', 'ut', 'vt', 'wa']
```

Comment:
this may have merge conflicts w/ master, as a PR (i think) turns this into a dictionary